djangocms_versioning/admin.py

```diff
@@ -110,6 +128,35 @@ def grouper_form_view(self, request):
         )
         return render(request, 'djangocms_versioning/admin/grouper_form.html', context)
 
+    def archive_view(self, request, pk):
+        """Archives the specified version and redirects back to the
+        version changelist
+        """
+        # FIXME: We should be using POST only for this, but some frontend
+        # issues need to be solved first. The code below just needs to
+        # be uncommented and a test is also already written (but currently
+        # being skipped) to handle the POST-only approach
+
+        # This view always changes data so only POST requests should work
+        # if request.method != 'POST':
+        #     raise Http404
+
+        version = self.model.objects.get(pk=pk)
```

Comment:
https://github.com/django/django/blob/master/django/contrib/admin/options.py#L1845-L1849

Reply:
I'm not really following what you want me to do or what that code does? There's no comments there as to what TO_FIELD_VAR actually means or does and didn't find docs on it either. And not sure in what context fields would be disallowed?